env_spec.py

```diff
@@ -0,0 +1,90 @@
+spec_str = 'DATABASE_URL\nADMIN_EMAIL\nDEBUG'
```

Comment:
I would move this declaration at the end inside the `if __name__ == '__main__'` statement, as it has no use outside of that scope.